package.json

```diff
@@ -16,7 +16,9 @@
   ],
   "activationEvents": [
     "onCommand:extension.sayHello",
-    "onView:servers"
+    "onView:servers",
+		"onLanguage:lua",
```

Comment:
Why do we need these two additional activation events?

Reply:
`"onLanguage:plaintext"` activation event is required because after installing VSIX. ssp-server output should be visible in vscode output channel.